packages/wonder-blocks-clickable/docs.md

```diff
@@ -1,4 +1,4 @@
-The Clickable component allows you to easily turn any custom compoent into a clickable one with consistent behavior and accesibility features.
+The Clickable component allows you to easily turn any custom component into a clickable one with consistent behavior and accesibility features.
```

Comment:
Nice find! I wonder how hard it would be to get some sort of spellchecker thing to run on our docs. Not something for you to tackle, just musing aloud.

Reply:
I wonder if it could be configured to only spell-check non-code parts of files and ignore words in the comments that are also in the code.